.github/workflows/tests-e2e.yml

```diff
@@ -202,6 +199,13 @@ jobs:
           echo "chromium_path=${CHROMIUM}" >> $GITHUB_ENV
         if: ( matrix.snapshots ) && ( github.event.pull_request.draft == false )
 
+      - name: Download AMP validator JS
+        id: amp_validator
+        run: |
+          cd $RUNNER_TEMP && curl -O -f -s -S https://cdn.ampproject.org/v0/validator_wasm.js
+          AMP_VALIDATOR_FILE="${RUNNER_TEMP}/validator_wasm.js"
+          echo "validator_file=$AMP_VALIDATOR_FILE" >> $GITHUB_ENV
```

Comment:
How is this different from `npm run pretest:js`? Should we use that here?

Reply:
That one is for unit tests, executed before the tests run

Reply:
To further clarify, the `pre<>` scripts are great for running tests locally, but on CI it's easier to just download the file in a separate step.